starterWorkflows/azure-kubernetes-service-helm.yml

```diff
@@ -84,12 +84,20 @@ jobs:
         tenant-id: ${{ secrets.AZURE_TENANT_ID }}
         subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
 
+      # Use kubelogin to configure your kubeconfig for Azure auth
+    - name: Set up kubelogin for non-interactive login
+      uses: azure/use-kubelogin@v1
+      with:
+        kubelogin-version: 'v0.0.25'
+
     # Retrieves your Azure Kubernetes Service cluster's kubeconfig file
     - name: Get K8s context
-      uses: azure/aks-set-context@v2.0
+      uses: azure/aks-set-context@v3
       with:
         resource-group: ${{ env.RESOURCE_GROUP }}
         cluster-name: ${{ env.CLUSTER_NAME }}
+        admin: 'false'
+        use-kubelogin: 'true'
 
     # Retrieves the credentials for pulling images from your Azure Container Registry
     - name: Get ACR credentials
```

Comment:
This is actually the wrong one too. It should be updated to be the same as the starter workflows. Using image pull secrets is less secure than just assuming they have the acr attached. We shouldn't need to use aks-set-context twice in each workflow.